### DISCUSSION TITLE
High res button not working for batches and Distilled CFG value not always respected for Flux

### DISCUSSION THREAD

**Discussion by anon0730** [lllyasviel/stable-diffusion-webui-forge#1184] [General]
I know Illyasviel is very busy mainly with backend development but there are some issues with the UI that hamper regular use for me, I thought about opening issues but there are over 500 open ones and I see the him frequent the discussions section instead of the issues.

First of all the high res button works for singular images but if you generated a batch of images and want to do the highres fix on one of them it does not work and gives the error `TypeError: tuple indices must be integers or slices, not str`. Here is a video:

https://github.com/user-attachments/assets/0b4f9f97-9e21-4add-8072-734a5ada5837

Secondly, the Distilled CFG Scale is not always being read by the backend it seems. Notice how I keep changing the value but only the first time the image changes. The value is not reported in the console log either. However, when I change the resolution of the image a little it then again reads the value from the Distilled CFG Scale field. Here is a video, it's long because I only have a 2070 Super, feel free to speed through the generation segments:

https://github.com/user-attachments/assets/0706dba1-92ad-4f35-adce-b612a6e8ecd2

**Comment by tutixinhzai** [comment]
I also got the same error" 
TypeError: tuple indices must be integers or slices, not str " in Forge . Can anyone help me?

**Reply by stripealipe** [reply]
Still waiting on a fix for 'distilled CFG' not working, or is there a workaround for now?! .... thanks
